tests/unit/test_tag_crazy_egg.py

```diff
@@ -19,32 +20,33 @@ class CrazyEggTagTestCase(TagTestCase):
 
     def test_tag(self):
         r = self.render_tag('crazy_egg', 'crazy_egg')
-        self.assertTrue('/1234/5678.js' in r, r)
+        assert '/1234/5678.js' in r
 
     def test_node(self):
         r = CrazyEggNode().render(Context())
-        self.assertTrue('/1234/5678.js' in r, r)
+        assert '/1234/5678.js' in r
 
     @override_settings(CRAZY_EGG_ACCOUNT_NUMBER=None)
     def test_no_account_number(self):
-        self.assertRaises(AnalyticalException, CrazyEggNode)
+        with pytest.raises(AnalyticalException):
```

Comment:
I was less sure about these ones, but went for it as I think it helps with consistency. Would be good to come back and add the message to all of these but don't think I'm up for volunteering for that at the moment.

Reply:
camelCase is for Java devs!  :nerd_face:  We're into snake_case, baby. :snake: :sunglasses: